docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -72,6 +72,9 @@ The following sections outline the commands that you can run against Grafana Mim
 
 The following commands interact with Grafana Mimir Alertmanager configuration and alert template files.
 
+> **Note:** The Mimir Alertmanager exposes the configuration API according to the path set by the `-server.path-prefix` flag. It does not use the path set by the `-http.alertmanager-http-prefix` flag.
+> With the default configuration of `-server.path-prefix`, the Alertmanager URL used as the `mimirtool` `--address` flag has no path portion.
```

Comment:
```suggestion
> If you run Mimir with the default configuration (`-server.path-prefix`, where the default value is `/`), then only set the hostname for the `mimirtool --address` flag; do not set a path-specific address. For example, `/` is correct, and `/alertmanager` is incorrect.
```

Reply:
we've used "`mimirtool` `--address` flag" in another article (alertmanager.md), should I update it there too?